Makefile

```diff
@@ -57,15 +63,29 @@ auth: ## Build threescale_wasm_auth filter.
 	make build
 	cp threescale-wasm-auth/target/wasm32-unknown-unknown/$(BUILD)/threescale_wasm_auth.wasm deployments/docker-compose/threescale_wasm_auth.wasm
 
+apisonator: ## Runs apisonator and redis container
+	docker run -p 6379:6379 -d --name my-redis redis --databases 2
+	docker run -e CONFIG_QUEUES_MASTER_NAME=redis://redis:6379/0 \
+            -e CONFIG_REDIS_PROXY=redis://redis:6379/1 -e CONFIG_INTERNAL_API_USER=root \
+            -e CONFIG_INTERNAL_API_PASSWORD=root -p 3000:3000 -d --link my-redis:redis \
+            --name apisonator quay.io/3scale/apisonator 3scale_backend start
+
 integration:
 	@echo "> Starting integration tests"
 	mkdir -p integration-tests/artifacts
 	cp deployments/docker-compose/cache_filter.wasm integration-tests/artifacts/cache_filter.wasm
 	cp deployments/docker-compose/singleton_service.wasm integration-tests/artifacts/singleton_service.wasm
 	cp deployments/docker-compose/threescale_wasm_auth.wasm integration-tests/artifacts/threescale_wasm_auth.wasm
 	go clean -testcache
+	docker run -p 6379:6379 -d --name my-redis redis --databases 2
```

Comment:
use apisonator as a prereq since you have already added a target for apisonator.

Reply:
Done